packages/styles/scss/components/content-item-horizontal-media/_content-item-horizontal-media.scss

```diff
@@ -0,0 +1,202 @@
+      margin-bottom: $carbon--layout-04;
+    }
+  }
+  :host(#{$dds-prefix}-content-item-horizontal-copy) ::slotted(:not([slot])),
+  .#{$prefix}--content-item-horizontal__item--copy {
+    margin-bottom: $carbon--spacing-07;
+    @include carbon--breakpoint(sm) {
+      margin-bottom: $carbon--spacing-07;
+    }
+    @include carbon--breakpoint(md) {
+      margin-bottom: $layout-05;
+    }
+    @include carbon--breakpoint(lg) {
+      margin-bottom: $layout-05;
+    }
+    p {
+      color: $text-01;
+    }
+  }
+
+  :host(#{$dds-prefix}-content-item-horizontal-media)
+    .#{$prefix}--content-item__cta,
+  .#{$prefix}--content-item-horizontal__item--cta {
+    margin-top: auto;
+
+    .#{$prefix}--link-list {
+      padding: 0;
+      &:first-of-type {
+        padding: 0;
+      }
+    }
+
+    .#{$prefix}--link-list li {
+      @include carbon--breakpoint(md) {
+        margin-bottom: $carbon--spacing-05;
+      }
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal__row {
+    @include carbon--make-row;
+
+    width: 100%;
+    min-height: 160px;
+
+    .#{$prefix}--content-item-horizontal__col {
+      flex-direction: column;
+
+      @include carbon--make-col-ready;
+
+      &:first-of-type {
+        @include carbon--breakpoint(sm) {
+          @include carbon--make-col(4, 4);
+        }
+
+        @include carbon--breakpoint(md) {
+          @include carbon--make-col(6, 8);
+        }
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(7, 10);
+        }
+      }
+
+      &:last-of-type {
+        @include carbon--breakpoint(sm) {
+          @include carbon--make-col(4, 4);
+
+          display: flex;
+        }
+
+        @include carbon--breakpoint(md) {
+          @include carbon--make-col(2, 8);
+
+          display: flex;
+        }
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(3, 10);
+
+          display: flex;
+        }
+      }
+    }
+
+    &.#{$prefix}--content-item-horizontal-media__align-right {
+      .#{$prefix}--content-item-horizontal__col {
+        flex-direction: column;
+
+        @include carbon--make-col-ready;
+
+        &:first-of-type {
+          @include carbon--breakpoint(sm) {
+            @include carbon--make-col(4, 4);
+
+            display: flex;
+          }
+
+          @include carbon--breakpoint(md) {
+            @include carbon--make-col(2, 8);
+
+            display: flex;
+          }
+
+          @include carbon--breakpoint(lg) {
+            @include carbon--make-col(3, 10);
+
+            display: flex;
+          }
+        }
+
+        &:last-of-type {
+          @include carbon--breakpoint(sm) {
+            @include carbon--make-col(4, 4);
+          }
+
+          @include carbon--breakpoint(md) {
+            @include carbon--make-col(6, 8);
+          }
+
+          @include carbon--breakpoint(lg) {
+            @include carbon--make-col(7, 10);
+          }
+        }
+      }
+    }
```

Comment:
In content patterns, we are under [the effort of transitioning to native CSS grid](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/4174). The styling strategy can be found at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.16.1/packages/web-components/.storybook/_simple-grid.scss#L12-L32 and https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.16.1/packages/web-components/.storybook/container.scss#L77-L141. Would you try...? Thanks!

Reply:
@asudoh I went ahead and refactored the styles to use CSS grid. This also allowed me to remove the conditional inside my `content-item-horizontal-media.ts` that was flipping the arrangement of the columns in the markup. The left/right layouts are controlled by the grid now which also makes it easier to place the image on top with the text on the bottom for mobile.